nodes/nomos-node-api/src/http/backend/axum.rs

```diff
@@ -273,15 +311,18 @@ where
         (status = 500, description = "Internal server error", body = String),
     )
 )]
-async fn add_cert(State(store): State<OverwatchHandle>, Json(cert): Json<Certificate>) -> Response {
+async fn add_cert(
+    State(handle): State<OverwatchHandle>,
+    Json(cert): Json<Certificate>,
```

Comment:
This handle should expect Cert that is serialized with `wire::serialize`. At least `mempool-da/add` had it in such way. 
Expecting bytes and then deserializing it with `wire::deserialize::<Certificate>(&payload).unwrap()` works, but I think this would ultimately need it's own extractor.

Reply:
Ah, thanks! I think this should be the reason why the CI continues failing.

Reply:
> tbh, if all other API are json we should be consistent and use json here as well unless there's a specific reason not to do so

Yes, make sense, I will try to make the CI works by using JSON args.